### PR TITLE
build: update teams channel connector

### PR DESCRIPTION
**Pull request by driskull**
build: update teams channel connector

### CODE REVIEW

File: TRAVIS_WEBHOOK_README.md [hunk 1]
```diff
@@ -1,18 +1,14 @@
 # Travis Webhook Instructions
 
-## Make sure travis is installed with gem
+## Make sure travis is installed with Homebrew
```
Comment (jcfranco) [inline TRAVIS_WEBHOOK_README.md:3]: Does this require the Homebrew version? It's only for macOS/Linux.

File: TRAVIS_WEBHOOK_README.md [hunk 2]
```diff
@@ -1,18 +1,14 @@
...
 
-`gem install travis`
+`brew install travis`
 
 ## Login to travis
 
-`travis login --pro`
+Create a personal access token on github.
```
Comment (jcfranco) [inline TRAVIS_WEBHOOK_README.md:9]: Nitpick: `GitHub`

Comment (jcfranco) [reply TRAVIS_WEBHOOK_README.md:9]: Maybe adding a link to generating a token would help: https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token

File: TRAVIS_WEBHOOK_README.md (hunk 1 above)
Comment (driskull) [reply TRAVIS_WEBHOOK_README.md:3]: Updated to use either. gem wasn't working correctly for me.

https://github.com/Esri/calcite-components/blob/master/TRAVIS_WEBHOOK_README.md

**Pull request merged**